independent-publisher-connectors/Shopify Admin Content/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@thatapiguy You can delete this file as it is not required for open sourcing your connector